src/Workleap.Extensions.Mongo.Ephemeral/ReusableMongoRunner.cs

```diff
@@ -33,12 +33,19 @@ public void Rent(Guid renter)
 
             // The lock and use count prevent multiple instances of local mongod processes for a same named MongoClient
             // that would degrade the overall performance.
-            this._runner ??= MongoRunner.Run(new MongoRunnerOptions
+            var options = new MongoRunnerOptions
             {
                 UseSingleNodeReplicaSet = true,
                 KillMongoProcessesWhenCurrentProcessExits = true,
-            });
+            };
 
+            var binaryDirectory = Environment.GetEnvironmentVariable("WORKLEAP_EXTENSIONS_MONGO_EPHEMERAL_BINARYDIRECTORY")?.Trim();
```

Comment:
Could we also do the same for arguments? I'm thinking of if we deploy an enterprise mongodb instance on our agents, then we would have to use `--storageEngine inMemory`, whereas maybe in local dev people would still use the regular WiredTiger from the community edition distributed through NuGet.

Reply:
I've added more env var